keep-ui/utils/hooks/useAlerts.ts

```diff
@@ -170,11 +170,20 @@ export const useAlerts = () => {
             new TextDecoder().decode(decompressedAlert)
           );
 
-          next(null, {
-            alerts: newAlerts,
-            lastSubscribedDate: new Date(),
-            isAsyncLoading: false,
-            pusherChannel,
+          next(null, (data) => {
+            if (data) {
+              return {
+                ...data,
+                alerts: [...newAlerts, ...data.alerts],
+              };
+            }
+
+            return {
+              alerts: newAlerts,
+              lastSubscribedDate: new Date(),
+              isAsyncLoading: false,
```

Comment:
If this event fires before the `async-done` event (and it always does, I believe), should we return `isAsyncLoading` as `false` or `true`? Since it's instantaneous, we won't be able to show the artificial loading

Reply:
Can we not emit it? This can be happening in 2 cases:
1. When the user is sending get /alerts (and then we asyncly fetch more alerts from providers until `async-done` is received)
2. When new alerts gets pushed "on the fly"